AWSTextractTests/AWSTextractTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+
+import XCTest
+import AWSTextract
+
+class AWSTextractTests : XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        AWSTestUtility.setupCognitoCredentialsProvider()
+    }
+    
+    func testAnalyzeDocumentInS3() {
+        let awsTextractClient = AWSTextract.default()
+        
+        let credentialsJson = AWSTestUtility.getCredentialsJsonAsDictionary()
+        let s3ObjectName = credentialsJson?["textract-s3-object-name"]
```

Comment:
All these optional unwraps without asserts means that if something low-level happens we'll never see a test failure until your force-unwrap of `analyzeDocumentRequest` in the method call below. Prefer `guard let` binding with an `XCTFail("<foo> unexpectedly nil")` message in the `else` clause so we can pinpoint the problems. (You could also force unwrap of course, but I'd prefer to avoid those even in test code.)

Reply:
Moved the assignments to guard statements.